dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -229,8 +211,11 @@ export const Discussion = ({
 					idApiUrl={idApiUrl}
 					page={commentPage}
 					setPage={setCommentPage}
-					filters={filters}
-					setFilters={setFilters}
+					filters={validFilters}
+					setFilters={(newFilters) => {
+						setHashCommentId(undefined);
```

Comment:
When they set filters, the user is no longer looking at a specific comment, so we can reset the `hashCommentId` value.